Makefile

```
@@ -1,10 +1,16 @@
+TS ?= tree-sitter
+
+build:
+	$(TS) generate --no-bindings
+
 download-examples: clean
-	./scripts/download-examples.sh
+	git clone --depth=1 https://github.com/nvim-treesitter/nvim-treesitter .tests/nvim-treesitter
```

```suggestion
	git clone --filter=blob:none --single-branch https://github.com/nvim-treesitter/nvim-treesitter .tests/nvim-treesitter
```

(This will give you a slim checkout but still a workable repo so you can bisect etc.)